test/uk/gov/hmrc/hecstubs/controllers/IndividualAccountOverviewControllerSpec.scala

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hecstubs.controllers
+
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{AnyContentAsEmpty, Result}
+import play.api.test.Helpers.{status, stubControllerComponents}
+import play.api.test.{FakeHeaders, FakeRequest}
+import play.api.test.Helpers._
+import uk.gov.hmrc.hecstubs.controllers.Testdata._
+
+import java.util.UUID
+import scala.concurrent.Future
+
+class IndividualAccountOverviewControllerSpec extends AnyWordSpec with Matchers {
+
+  lazy val mockCC = stubControllerComponents()
+
+  def fakeRequest(environment: String, correlationId: String) = FakeRequest(
+    method = "GET",
+    uri = "/",
+    headers = FakeHeaders(
+      Seq(("Content-type", "application/json"), ("Environment", environment), ("CorrelationId", correlationId))
```

Comment:
you can probably get away with not having content type here - there shouldn't be any I don't think cos there is no body in the request

Reply:
updated